dap-etl/extraction/src/test/scala/org/broadinstitute/monster/dap/common/OkWrapperSpec.scala

```diff
@@ -0,0 +1,52 @@
+package org.broadinstitute.monster.dap.common
+
+import okhttp3.Request
+import okhttp3.mockwebserver.{MockResponse, MockWebServer}
+import org.broadinstitute.monster.dap.common.OkWrapper.MAX_RETRIES
+import org.scalatest.flatspec.FixtureAsyncFlatSpec
+import upack.Int64
+
+class OkWrapperSpec extends FixtureAsyncFlatSpec {
+
+  behavior of "OkWrapper"
+
+  type FixtureParam = MockWebServer
+
+  override def withFixture(test: OneArgAsyncTest) = {
+    val server = new MockWebServer
```

Comment:
This is provided by okhttp and let's us setup  full mock web server, similar to our redcapclient mocking (if we had the time, I'd say we should scrap our homegrown implementation and move to this).